ui/ui-showcase.jsx

```diff
@@ -760,6 +768,19 @@
 
             <FramedExample cssClass="fx-embedded-panel"
                            dashed={true}
+                           height={250}
+                           summary="Share panel"
```

Comment:
If in fake-mozLoop.js you add this in the GetLoopPref function, the facebook button should display as well:

```js
        case "facebook.enabled":
          return true;
```